src/main/java/org/amahi/anywhere/activity/ServerFilesActivity.java

```diff
@@ -555,4 +553,22 @@ private void tearDownFilesState(Bundle state) {
         private State() {
         }
     }
+
+    @SuppressLint("ValidFragment")
```

Comment:
Why did you suppress the lint? It's giving a fair warning:
![image](https://user-images.githubusercontent.com/15021142/31985314-48caa0cc-b982-11e7-8928-393092a17f6e.png)

Also the app **crashes** on screen rotation with this in the logs:
` java.lang.RuntimeException: Unable to start activity ComponentInfo{org.amahi.anywhere/org.amahi.anywhere.activity.ServerFilesActivity}: android.app.Fragment$InstantiationException: Unable to instantiate fragment org.amahi.anywhere.activity.ServerFilesActivity$ProgressDialogFragment: make sure class name exists, is public, and has an empty constructor that is public.`